src/NuGet.Services.AzureSearch/SearchService/Models/StatusResponse.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.Services.AzureSearch.SearchService
+{
+    public class StatusResponse
+    {
+        public bool Success { get; set; }
```

Comment:
What does the Success mean? Is it correlated with the values below?

Reply:
I'll leave a code comment. Success means that all of the search service dependencies could be interacted with successfully. This essentially means that the search and hijack indexes can be queried and that the auxiliary data was loaded at least once (stale data is okay).